iina/Base.lproj/Localizable.strings

```diff
@@ -11,78 +11,78 @@
 "off" = "Off";
 
 // PlaylistViewController.swift
-"playlist.playlist_cap" = "PLAYLIST";
-"playlist.chapter_cap" = "CHAPTERS";
+"playlist.playlist_cap" = "LISTE DE LECTURE";
```

Comment:
Apple uses the term "playlist" in the french iTunes, maybe we should stick with it ?